Cython/Utility/ObjectHandling.c

```diff
@@ -1421,10 +1458,18 @@ static CYTHON_INLINE PyObject *__Pyx__GetModuleGlobalName(PyObject *name)
         return result;
     }
 #else
+#if !CYTHON_AVOID_THREAD_UNSAFE_BORROWED_REFS
     result = PyDict_GetItem($moddict_cname, name);
+#else
+    PyDict_GetItemRef($moddict_cname, name, &result);
```

Comment:
This function is new in Py3.13a1 and needs a version guard.
See https://github.com/python/cpython/commit/41ca16455188db806bfc7037058e8ecff2755e6c